CHANGES.md

```diff
@@ -72,6 +72,15 @@
 - improve media type parsing (@seliopou #542, @dinosaure #725)
 - add comparison functions for Request.t and Response.t via ppx_compare (@msaffer-js @dinosaure #686)
 
+## v2.5.5 (2021-03-15)
+
+- `Cohttp_async.resolve_local_file`, `Cohttp_lwt.resolve_local_file` and `Cohttp_lwt_unix.resolve_file`
+  are now the same code under the hood (`Cohttp.Path.resolve_local_file`). The old names
+  have been preserved for compatibility, but marked as deprecated. This changes the behavior of
```

Comment:
Instead of marked as deprecated I will modify this as “will be marked as deprecated in the next released” if that is fine